### ISSUE TITLE
Keymap for ^ on GERMAN_MAC

### ISSUE THREAD

**Issue by Piggeldi2013** [trustcrypto/OnlyKey-Firmware#157]
The keymap for the ^ sign on Keyboard Layout GEMAN_MAC doesn't seem to work correctly.

If used in a password, the ^ sign is replaced with a space.

This seems to be the only thing that isn't working in the keymap.

So, given a password of: 
asd^dde
results in:
asd dde

Setting of the password has been tested with the App and also on command line - with same results.

Please advise.

**Comment by onlykey** [comment]
@Piggeldi2013 Can you use the app here and let us know what keycode you are getting when you type ^ with your keyboard and then what keycode you get when OnlyKey types this?

https://apps.apple.com/us/app/key-codes/id414568915?mt=12

**Comment by Piggeldi2013** [comment]
<img width="249" alt="image" src="https://user-images.githubusercontent.com/8721631/198676368-a13e37d5-fbec-4abe-96f9-23554418fa1d.png">

Same for both - adding it in the App, and also with cli.

**Comment by onlykey** [comment]
No I mean if you type the ^ character via your keyboard like this:
![image](https://user-images.githubusercontent.com/18121756/198678098-37b67a6b-0497-4daa-9d97-49757e9239cf.png)

Then 2nd screenshot of if OnlyKey types ^ (set your password as this in a slot for the test)

**Comment by Piggeldi2013** [comment]
Thanks for coming back.

The key (pressed at keyboard) is:
<img width="223" alt="image" src="https://user-images.githubusercontent.com/8721631/198680240-f479a205-4cf7-4e29-af1a-d69481c4710c.png">

And the image in my post before shows how the "password" is sent by the OnlyKey.

**Comment by onlykey** [comment]
Can you send the whole thing like, type "^ ^" via keyboard and OnlyKey like this:

^ ^ typed on keyboard:
![image](https://user-images.githubusercontent.com/18121756/198697385-723e07cc-58e9-4c78-9287-1c97b54175b4.png)


^ ^ typed via OnlyKey
![image](https://user-images.githubusercontent.com/18121756/198697749-7e303b79-3660-4167-b8d7-491e9741428a.png)

**Comment by Piggeldi2013** [comment]
Sure I can - here the screens for ^ ^

Keyboard: 
<img width="372" alt="image" src="https://user-images.githubusercontent.com/8721631/198817373-cd71972d-918a-4f89-ab32-045718c85dd1.png">

OnlyKey:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/8721631/198817382-cc88e33b-5ef5-4e17-9d71-133458cd0627.png">

**Comment by onlykey** [comment]
@Piggeldi2013 Oh sorry it looks like you don't have the "Log key up events and modifier changes" box checked, can you send the same results again with that box checked?

**Comment by Piggeldi2013** [comment]
Hi, no problem - I'm glad that I get help and can give some more infos - thanks for that!!

Keyboard:
<img width="380" alt="image" src="https://user-images.githubusercontent.com/8721631/198961557-ff6dd2ec-9c7a-40a1-af50-0ec0530d2b14.png">

OnlyKey:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/8721631/198961688-839bf28b-de68-4348-90ea-2d0903fe4b29.png">

**Comment by Piggeldi2013** [comment]
Any news on that? Something I can help or provide?

**Comment by onlykey** [comment]
@Piggeldi2013 No, thank you for the help. It looks like we can generate the ^ character using the combination of SHIFT+ALT+6

We will get this change into the next release

**Comment by onlykey** [comment]
@Piggeldi2013 Can you please test the firmware here? I believe the issue is fixed now.
[Signed_OnlyKey_3_0_4_STD.txt](https://github.com/trustcrypto/OnlyKey-Firmware/files/10170347/Signed_OnlyKey_3_0_4_STD.txt)
